algorithms/indexing/lattice_search/low_res_spot_match.py

```diff
@@ -117,9 +117,25 @@ def __ne__(self, other):
 
 
 class LowResSpotMatch(Strategy):
-    """Lattice search by matching low resolution spots to candidate indices
-    based on resolution and reciprocal space distance between observed spots.
     """
+    A lattice search strategy matching low resolution spots to candidate indices.
+
+    The match is based on resolution and reciprocal space distance between observed
+    spots.  A prior unit cell is required and solutions are assessed by matching the
```

Comment:
It turns out it wants the symmetry too (to construct the library of candidate reflections for matching)
```suggestion
    spots. A prior unit cell and space group are required and solutions are assessed by matching the
```